Clients/src/presentation/pages/Home/index.tsx

```diff
@@ -1,5 +1,7 @@
+import ProjectsOverview from "../../components/ProjectsOverview";
+
 const Home = () => {
```

Comment:
You did not need to nest the components that were suppose to be inside Home page (with the url "/") in another component.
Please move what you have inside `<ProjectsOverview />` to the Home page and then, if there's any component or widgets that we don't have among our components, then, we can create a new component.